packages/fuels-accounts/src/kms/aws/client.rs

```diff
@@ -0,0 +1,63 @@
+pub use aws_config::{
+    default_provider::credentials::DefaultCredentialsChain, BehaviorVersion, Region, SdkConfig,
+};
+pub use aws_sdk_kms::config::Credentials;
+use aws_sdk_kms::Client;
+
+#[derive(Clone)]
+pub struct AwsConfig {
+    sdk_config: SdkConfig,
+}
+
+impl AwsConfig {
+    pub async fn from_environment() -> Self {
+        let loader = aws_config::defaults(BehaviorVersion::latest())
+            .credentials_provider(DefaultCredentialsChain::builder().build().await);
+
+        Self {
+            sdk_config: loader.load().await,
+        }
+    }
+
+    #[cfg(feature = "test-helpers")]
+    pub async fn for_testing(
+        credentials: Credentials,
+        region: Region,
+        endpoint_url: String,
+    ) -> Self {
+        let sdk_config = aws_config::defaults(BehaviorVersion::latest())
+            .credentials_provider(credentials)
+            .endpoint_url(endpoint_url)
+            .region(region)
+            .load()
+            .await;
+
+        Self { sdk_config }
+    }
+
+    pub fn endpoint_url(&self) -> Option<&str> {
+        self.sdk_config.endpoint_url()
+    }
+
+    pub fn region(&self) -> Option<&Region> {
+        self.sdk_config.region()
+    }
+}
+
+#[derive(Clone, Debug)]
+pub struct AwsClient {
+    client: Client,
+}
```

Comment:
Consider dropping this wrapper as well, since we're reexporting from the aws crates.